scripts/r.in.wms/r.in.wms.py

```diff
@@ -227,6 +241,12 @@ def main():
         wms.GetCapabilities(options)
     else:
         from wms_base import GRASSImporter
+        # set proxy
```

Comment:
Suggestion: add warning that proxy options will be ignored (if given) when using different driver from gdal